src/tfr_control/arduino/quadrature.h

```diff
@@ -53,11 +53,37 @@ class Quadrature
       double t_1 = 0;
     };
 
-    const double CPR;
+    const int CPR;
 
     EncoderData data;
 
     Encoder encoder;
     
 };
+/*
+  Class for efficiently reading quadrature encoder data, instantiate, and
+  call getVelocity in the main loop. This is designed to be used in a system 
+  who only has one job, to read these quadrature encoders
```

Comment:
And now the system has more than one job, hahahaha: is this comment still relevant?